Source/Core/Core/NetworkCaptureLogger.cpp

```diff
@@ -145,13 +127,13 @@ void PCAPSSLCaptureLogger::Log(LogType log_type, const void* data, std::size_t l
 
   if (getsockname(socket, reinterpret_cast<sockaddr*>(&sock), &sock_len) != 0)
   {
-    RestoreState(state);
+    Common::RestoreNetworkErrorState(state);
```

Comment:
Nitpickin' but this could also go in a `ScopedGuard` so that instead of 3 lines, there's just one.